includes/class-experimental-abtest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * A class that interacts with Explat A/B tests.
```

Comment:
Should we add some comments here with the source URL this was copied from and any notes about when we can remove this in the future (if applicable)?